muted-tests.yml

```diff
@@ -567,8 +567,6 @@ tests:
 - class: org.elasticsearch.xpack.inference.qa.mixed.CohereServiceMixedIT
   method: testCohereEmbeddings
   issue: https://github.com/elastic/elasticsearch/issues/130010
-- class: org.elasticsearch.xpack.esql.qa.mixed.MixedClusterEsqlSpecIT
-  issue: https://github.com/elastic/elasticsearch/issues/128224
```

Comment:
Thanks! I think this change will not fix this issue and it's probably not safe to unmute the suite, because the `dense_vector` dataset seems to be problematic on main, not `colors`.

I have a PR that should make the mute more targeted though: https://github.com/elastic/elasticsearch/pull/130089